app/uk/gov/hmrc/exports/connectors/CustomsDataStoreConnector.scala

```diff
@@ -17,21 +17,20 @@
 package uk.gov.hmrc.exports.connectors
 
 import javax.inject.Inject
-
-import scala.concurrent.{ExecutionContext, Future}
-
+import play.api.http.Status.NOT_FOUND
 import uk.gov.hmrc.exports.config.AppConfig
 import uk.gov.hmrc.exports.models.VerifiedEmailAddress
-import play.api.http.Status.NOT_FOUND
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, Upstream4xxResponse}
 import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, Upstream4xxResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
 
 class CustomsDataStoreConnector @Inject()(http: HttpClient)(implicit appConfig: AppConfig, ec: ExecutionContext) {
 
   import CustomsDataStoreConnector._
 
   def getEmailAddress(eori: String)(implicit hc: HeaderCarrier): Future[Option[VerifiedEmailAddress]] =
-    http.GET[VerifiedEmailAddress](verifiedEmailUrl(eori)).map(Some(_)).recover {
+    http.GET[VerifiedEmailAddress](verifiedEmailUrl(eori)).map(Option(_)).recover {
```

Comment:
Out of curiosity, what is the significance of this change?

Reply:
Actually, I realized it can be simplified with `http.GET[Option[VerifiedEmailAddress]]`. I just wanted to avoid having empty `Some`. But I am not sure if it might happen.

Reply:
It cannot happen. The HttpReads raises an exception if it cannot instantiate the class. IMO the `Option` here is not good practice... practically you are saying that the library could return `null`.

Reply:
If you replace the placeholder `_` with a proper identifier you will notice that its type is `VerifiedEmailAddress`, which of course cannot be `null`.